erpnext/agriculture/doctype/harvest/harvest.py

```diff
@@ -12,12 +12,30 @@ class Harvest(Document):
 	def on_submit(self):
 		self.create_integration_request()
 
+		plants = self.get('plants')
+		for p in plants:
+			plant = frappe.get_doc("Plant",p.plant)
+			if self.harvest_type == "Full Harvest":
+				frappe.db.set_value("Plant", plant.name, "completely_harvested", True)
+
+	def on_cancel(self):
+		plants = self.get('plants')
+		for p in plants:
+			plant = frappe.get_doc("Plant",p.plant)
+			if self.harvest_type == "Full Harvest":
+				frappe.db.set_value("Plant", plant.name, "completely_harvested", False)
```

Comment:
frappe.db.set_value("Plant", plant.plant, "completely_harvested", False)

Reply:
fixed this with the latest commit